_includes/components/bio--partner.html

```diff
@@ -0,0 +1,8 @@
+<div class="partner-bio bio-list__bio">
+  <div class="partner-bio__image">
+    {% picture partner_square partners/{{ include.partner.slug }}/{{ include.partner.slug }}-square-outside.jpg alt="{{ include.partner.name }}" %}
```

Comment:
This seems to assume too much of the partner. In most cases, they wouldn't have a "square outside" version of a picture, so this should be more generalized - and then the markdown file should be more straightforward about how to properly point to the correct file